public/app/plugins/datasource/mysql/partials/config.html

```diff
@@ -23,6 +23,7 @@ <h3 class="page-heading">MySQL Connection</h3>
         on-reset="ctrl.onPasswordReset"
         on-change="ctrl.onPasswordChange"
         inputWidth="9"
+		aria-label="'Password'"
```

Comment:
this is really just nitpicking but can you please align it with the other attributes?